PrebidMobile/PrebidMobileRendering/ORTB/PBMORTBBidRequestExtPrebid.m

```diff
@@ -40,6 +47,13 @@ - (nonnull PBMJsonDictionary *)toJsonDictionary {
     
     PBMMutableJsonDictionary * const storedRequest = [PBMMutableJsonDictionary new];
     ret[@"storedrequest"] = storedRequest;
+    NSArray *renderers = [[PrebidMobilePluginRegister shared] getAllPlugins];
+//    TODO How to get the isOriginalAPI value here
+//    if (renderers.count != 0 || self.adConfiguration.isOriginalAPI) {
```

Comment:
`PBMParameterBuilderService` contains `adConfiguration`. 

The most straightforward way is to pass `adConfiguration` to  `[PBMORTBParameterBuilder buildOpenRTBFor:bidRequest];` and further through the call chain.

Reply:
@github-richard-depierre, just want to remind you that `isOriginalAPI` is not utilized in the logic yet. Please, see my above comment about how to get this value.